### ISSUE TITLE
Certain users located in Australia unable to access apps.risevision.com

### ISSUE THREAD

**Issue by kaden-rise** [Rise-Vision/rise-vision-apps#1534]
**Describe the bug**
Certain users located in Australia are unable to access apps.risevision.com. The site won't load for them at all. 

**Reach**
So far 2 users, but could be affecting multiple users located in Australia.

**Impact**
It happens every time the user tries to navigate to apps.risevision.com. One of the affected users has Optus as their ISP. They were able to resolve the issue by changing their DNS servers to 8.8.8.8 so it may be an ISP DNS problem.

**To Reproduce**
I'm unable to reproduce this behavior.

**Comment by ezequielc** [comment]
@kaden-rise The first hypothesis around theses cases are firewall blocks. Just want to make sure that this is not the case.
Also, I would like more details on this. Are the two users from different companies? Are they from different cities? 
And, do you know if it used to work and stop suddenly? I am wondering if it could be an intermittent problem or if they still can't access. Thanks

**Comment by kaden-rise** [comment]
@ezequielc The users in question are from different companies. I don't know if the users are in the same cities, but as of today I confirmed that both users have the same ISP which is Optus. I also confirmed that both users were able to resolve the issue by changing their DNS server to 8.8.8.8. Both users contacted me on Monday so I would imagine it was working just fine until then.

**Comment by ezequielc** [comment]
@kaden-rise I did some research and it seems that other Optus customers are having similar issues with overseas traffic ([source](https://yescrowd.optus.com.au/t5/Broadband-Telephony/Australian-DNS/td-p/571810)). Their solution was to set 8.8.8.8 as DNS as well.
Also, I checked Apps traffic from Australia and it looks normal. But I did notice that we used to have 3-5 daily access from Optus customers and since 4-Feb we are no longer getting traffic from them.

To be clear, I don't think there is anything we can do from our end. I would ask the users to contact Optus support. @alex-deaconu @alanclayton Let me know if you disagree.
Thanks!
